src/compiler/parser.ts

```diff
@@ -3454,7 +3461,7 @@ namespace Parser {
             entity = finishNode(
                 factory.createQualifiedName(
                     entity,
-                    parseRightSideOfDot(allowReservedWords, /* allowPrivateIdentifiers */ false) as Identifier
+                    parseRightSideOfDot(allowReservedWords, /* allowPrivateIdentifiers */ false, /** allowUnicodeEscapeSequenceInIdentifierName */ true) as Identifier

```

Comment:
Nit, but all instances of `/** allowUnicodeEscapeSequenceInIdentifierName */` should be `/*allowUnicodeEscapeSequenceInIdentifierName*/` for consistency with the rest of the codebase.